cli/js/process.ts

```diff
@@ -99,17 +100,19 @@ export class Process<T extends RunOptions = RunOptions> {
   }
 }
 
-export type ProcessStatus =
-  | {
-      success: true;
-      code: 0;
-      signal?: undefined;
-    }
-  | {
-      success: false;
-      code: number;
-      signal?: number;
-    };
+type ProcessStatusSuccessful = {
+  success: true;
+  code: 0;
+  signal?: undefined;
+};
+
+type ProcessStatusUnsuccessful = {
+  success: false;
+  code: number;
+  signal?: number;
+};
+
+export type ProcessStatus = ProcessStatusSuccessful | ProcessStatusUnsuccessful;
```

Comment:
It's helpful for this to just match the structure of the definition in `lib.deno.ns.ts`. Please revert.